RPIPs/rpip-draft_rewards_tree.md

```diff
@@ -0,0 +1,36 @@
+---
+rpip: #<to be assigned>
+title: Rewards Tree Specification
+description: Specifies how RPL Inflation and Smoothing Pool rewards are allocated to node operators
+author: Ramana Kumar (@xrchz), Joe Clapis (@jcrtp)
+discussions-to: https://github.com/rocket-pool/rocketpool-research/issues
+status: Draft
+type: Informational
+created: 2024-04-09
+---
+
+## Abstract
+This RPIP describes the current state of the specification of the Rocket Pool rewards system (first introduced in [Redstone](https://medium.com/rocket-pool/rocket-pool-the-merge-redstone-601d9efd6b4)) for node operators, wherein ETH and RPL rewards are allocated at regular intervals to staking nodes.
+
+## Motivation
+Rocket Pool's allocation of rewards to node operators needs a clear specification that can be independently implemented and verified so that node operators know what to expect and have a basis for recourse in case of any misallocation.
```

Comment:
"and have a basis for recourse in case of any misallocation" is not currently implied anywhere and pretty significant. I'd suggest simply removing or maybe "know what to expect and are able to verify allocations." or something.

Reply:
For the future Core Protocol version I was hoping the legitimacy from being a ratified RPIP would be the basis for recourse. Are you objecting to that, or just to it being present in the Informational version?

Reply:
(after chat elsewhere) rewording as suggested